components/Editor/index.tsx

```diff
@@ -0,0 +1,230 @@
+import React, { useEffect } from 'react';
+
+import {
+  Remirror,
+  useRemirror,
+  useRemirrorContext,
+  EditorComponent,
+  EditorState,
+  useChainedCommands,
+  useActive,
+} from '@remirror/react';
+import { prosemirrorNodeToHtml } from '@remirror/core-utils';
+
+import {
+  BoldExtension,
+  ItalicExtension,
+  UnderlineExtension,
+  LinkExtension,
+  BulletListExtension,
+  OrderedListExtension,
+  AnnotationExtension,
+  TableExtension,
+} from 'remirror/extensions';
+
+import 'remirror/styles/all.css';
+import './Editor.scss';
+
+import { DmpIcon } from '/components/Icons';
+import {
+  Button,
+  Group,
+  Separator,
+  ToggleButton,
+  Toolbar,
+  Form,
+  Menu,
+  MenuItem,
+  MenuTrigger,
+  Popover,
+} from 'react-aria-components';
+
+
+const AnnotationButton: React.FC = () => {
+  const { helpers, commands, getState } = useRemirrorContext({ autoUpdate: true });
+
+  const handleAnnotation = () => {
+    const selectedText = getState().selection;
+
+    if (selectedText) {
+      const annotation = prompt('Enter your annotation:', '');
+      if (annotation) {
+        commands.addAnnotation({ text: selectedText, annotation });
+      }
+    }
+  };
+
+  return (
+    <Button onPress={handleAnnotation}>
+      <DmpIcon icon="message" />
+    </Button>
+  )
+}
+
+
+const TableGroup: React.FC = () => {
+  const chain = useChainedCommands();
+  const { helpers, commands, getState } = useRemirrorContext({ autoUpdate: true });
+
+  return (
+    <Group aria-label="Table Tools">
+      <MenuTrigger>
+        <Button aria-label="Menu">
+          <DmpIcon icon="table" />
+        </Button>
+        <Popover>
+          <Menu>
+            <MenuItem onAction={() => commands.createTable()}>Insert Table</MenuItem>
```

Comment:
Very minor, but can we change this to "Add Table" since we're using that verb for the columns and rows